addons/account/models/res_config_settings.py

```diff
@@ -32,6 +32,22 @@ class ResConfigSettings(models.TransientModel):
     group_warning_account = fields.Boolean(string="Warnings in Invoices", implied_group='account.group_warning_account')
     group_cash_rounding = fields.Boolean(string="Cash Rounding", implied_group='account.group_cash_rounding')
     group_fiscal_year = fields.Boolean(string='Fiscal Years', implied_group='account.group_fiscal_year')
+    # group_show_price_subtotal and group_show_price_total are opposite,
+    # so we can assume exactly one of them will be set, and not the other.
+    # We need both of them to coexist so we can take advantage of automatic group assignation.
+    group_show_price_subtotal = fields.Boolean(
+        "Show subtotal",
+        implied_group='account.group_show_price_subtotal',
+        group='base.group_portal,base.group_user,base.group_public')
+    group_show_price_total = fields.Boolean(
+        "Show total",
+        implied_group='account.group_show_price_total',
+        group='base.group_portal,base.group_user,base.group_public')
+    sale_show_tax = fields.Selection([
```

Comment:
I propose document_show_tax to match the name change we did in the settings.

Reply:
We also had an issue with the name of the groups group_show_price_total and group_show_price_subtotal because they don't really convey their meaning.

Maybe we need to think about a better name for all 3.